src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViews.contribution.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Registry } from 'vs/platform/registry/common/platform';
+import { localize } from 'vs/nls';
+import { IConfigurationRegistry, Extensions as ConfigExtensions } from 'vs/platform/configuration/common/configurationRegistry';
+
+export const NOTEBOOK_VIEWS_ENABLED_PROPERTY = 'notebookViews.enabled';
+
+const properties = {};
+properties[NOTEBOOK_VIEWS_ENABLED_PROPERTY] = {
+	'type': 'boolean',
+	'default': false,
+	'description': localize('notebookViews.enabled', "(Preview) enable notebook views")
+};
+
+const configurationRegistry = Registry.as<IConfigurationRegistry>(ConfigExtensions.Configuration);
+
+configurationRegistry.registerConfiguration({
+	'id': 'notebookViews',
```

Comment:
Why is this a top-level configuration instead of something under the existing notebooks section? https://github.com/microsoft/azuredatastudio/blob/main/src%2Fsql%2Fworkbench%2Fcontrib%2Fnotebook%2Fbrowser%2Fnotebook.contribution.ts#L191

Reply:
Notebook views are intended to be the replacement for XTS views. We are planning on adding a few more settings related to the views, such as enabling/disabling automatic layout generation, and I thought it made a little bit more sense to group them all instead of mixing them with the existing notebook section.

Reply:
mmm, I think it makes more sense to have any settings related to Notebooks under one configuration section. @chlafreniere thoughts here?